### PR TITLE
Static isFinite(value) utility

### PR DESCRIPTION
**Pull request by unkillbob**
```js
VendNumber.isFinite(null) // false
VendNumber.isFinite('') // false
VendNumber.isFinite(Infinity) // false

VendNumber.isFinite(0) // true
VendNumber.isFinite(Number.MAX_VALUE) // true
VendNumber.isFinite('-123.456') // true
```

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -387,4 +387,43 @@ describe('VendNumber', () => {
...
+      expect(isFinite(undefined)).to.be.false
+      expect(isFinite(null)).to.be.false
+      expect(isFinite(NaN)).to.be.false
+      expect(isFinite(false)).to.be.false
+      expect(isFinite('')).to.be.false
+      expect(isFinite('not a number')).to.be.false
+    })
+
+    it('should return true for finite numbers represented as strings', () => {
+      expect(isFinite('123.45')).to.be.true
```
Comment (chrisgreeff) [inline test/test.js]: test for `0`?

**Comment by chrisgreeff** [comment]
1 small comment, otherwise LGTM

**Comment by unkillbob** [comment]
@chrisgreeff https://github.com/vend/vend-number/commit/8ff289ce92a4ba2d80f9bf04b5da1a69c3c14b75 how 'bout now?

**Comment by chrisgreeff** [comment]
Merge away!

**Pull request merged**